srv/wordpress/wp-content/plugins/acore-wp-plugin/src/components/AdminPanel/Settings.view.php

```diff
@@ -289,6 +289,114 @@ public function getSettingsRender() {
+                    </tr>
+                    <tr>
+                        <th>
+                            <label for="acore_db_eluna_port">Database Eluna Port:</label>
+                        </th>
+                        <td>
+                            <input type="text" name="acore_db_eluna_port" value="<?= Opts::I()->acore_db_eluna_port; ?>" size="20">
+                        </td>
+                    </tr>
+                    <tr>
+                        <th>
+                            <label for="acore_db_eluna_user">Database Eluna User:</label>
+                        </th>
+                        <td>
+                            <input type="text" name="acore_db_eluna_user" value="<?= Opts::I()->acore_db_eluna_user; ?>" size="20" >
+                        </td>
+                    </tr>
+                    <tr>
+                        <th>
+                            <label for="acore_realm_alias">Database Eluna Pass:</label>
+                        </th>
+                        <td>
+                            <input type="password" name="acore_db_eluna_pass" value="<?= Opts::I()->acore_db_eluna_pass; ?>" size="20" >
+                        </td>
+                    </tr>
+                    <tr>
+                        <th>
+                            <label for="acore_db_eluna_name">Database Eluna Name:</label>
+                        </th>
+                        <td>
+                            <input type="text" name="acore_db_eluna_name" value="<?= Opts::I()->acore_db_eluna_name; ?>" size="20" >
+                        </td>
+                    </tr>
+                    <tr>
+                </tbody>
+            </table>
+
+            <hr />
+
+            <table class="form-table" role="presentation">
+                <tbody>
+                    <tr>
+                        <th scope="row">
+                        <label>Eluna modules:</label>
+                        </th>
+                    </tr>
+                    <tr>
+                        <th>
+                            <label for="eluna_recruit_a_friend">Recruit a Friend <a href="https://github.com/55Honey/Acore_RecruitAFriend" target="_blank"><span class="dashicons dashicons-external"></span></a></label>
+                        </th>
+                        <td>
+                            <select name="eluna_recruit_a_friend">
+                                <option value="0">Disabled</option>
+                                <option value="1" <?php if (Opts::I()->eluna_recruit_a_friend == '1') echo 'selected'; ?>>Enabled</option>
+                            </select>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+            <hr />
+            <p class="submit">
+                <input type="submit" name="Submit" class="button-primary" value="<?php esc_attr_e('Save Changes', Opts::I()->page_alias) ?>" />
+            </p>
+
+        </form>
+        </div>

```

Comment:
what are you closing here?

Reply:
The div opened by the echo in the beginning of getElunaSettingsRender